backwards-compatability/node8/index.ts

```diff
@@ -0,0 +1,6 @@
+import {NodeSDK, api} from '@opentelemetry/sdk-node';
+
+const sdk = new NodeSDK();
```

Comment:
If  you start the sdk with no options, no tracer or meter provider is registered.